DataFormats/MuonReco/interface/MuonCSCRecHitCluster.h

```diff
@@ -0,0 +1,53 @@
+#ifndef DataFormats_MuonCSCRecHitCluster_h
+#define DataFormats_MuonCSCRecHitCluster_h
+
+#include <vector>
+#include "DataFormats/Common/interface/SortedCollection.h"
+
+#include "DataFormats/CSCRecHit/interface/CSCRecHit2DCollection.h"
+#include "DataFormats/MuonReco/interface/MuonRecHitCluster.h"
+
+namespace reco {
+
+  class MuonCSCRecHitCluster : public MuonRecHitCluster {
```

Comment:
Is there any chance `MuonCSCRecHitCluster` would be destructed via a pointer to base class?

Reply:
The two classes has been merged into a single one for now.